examples/js/exporters/GLTFExporter.js

```diff
@@ -325,6 +325,26 @@ THREE.GLTFExporter.prototype = {
 
 		}
 
+		/**
+		 * Serializes a userData.
+		 *
+		 * @param {Object} userData
+		 * @returns {Object}
+		 */
+		function serializeUserData( userData ) {
+
+			try {
+
+				return JSON.parse( JSON.stringify( userData ) );
+
+			} catch ( error ) {
+
+				throw new Error( 'THREE.GLTFExporter: userData can\'t be serialized' );
```

Comment:
I just copied. I don't have strong preference.

@fernandojsg You seemed to add the original this try/catch code. Any special reasons why we should halt if we fail to serialize userData?

Reply:
@donmccurdy do you mean to keep the try/catch but using a `console.warn` instead of an error? or just remove the whole try/catch.
If the first I totally agree that will be fine. Regarding removing the try/catch, I would prefer to leave it there as `userData` could be everything and it could break on `stringify`

Reply:
If you didn't have any special reasons why it halted, try-catch with console.warn would sound reasonable for me.  Agreed with

> Loss of userData doesn't seem to me like it needs to halt export.

    try {

        return JSON.parse( JSON.stringify( userData ) );

    } catch ( error ) {

        console.warn( 'THREE.GLTFExporter: userData won\'t be serialized ' +
                      'because JSON.stringify failed. ' + error.message );
        return {};

    }

Reply:
Yep that looks fine for me! :+1: